### DISCUSSION TITLE
The best way to contribute landscapes?

### DISCUSSION THREAD

**Discussion by davidneedham** [endless-sky/endless-sky#8573] [General]
Hello! I'm a fan of the game and have been looking for the best way for me to contribute back. I read through the [Contributing section of the Wiki](https://github.com/endless-sky/endless-sky/wiki/HelpWanted) and I'd like to contribute landscapes to reduce the number of duplicates reported in #7107.

I found the [style guide for landscapes](https://github.com/endless-sky/endless-sky/wiki/StyleGoals#landscapes), but I don't see any documentation about the process to contribute. Is there a wiki or spreadsheet to update saying that I'm working on something, or that my PR has been merged and is no longer a dupe? Should I create a PR for each landscape, or is there a review process to step through before that?

Thanks!

**Comment by Hecter94** [comment]
Hey there! We actually just had someone contribute a landscape image to remove one of those duplicates, which you can find here for reference: https://github.com/endless-sky/endless-sky/pull/8550.

Generally speaking, that would be the best method for contributing. 
Find a duplicate on the list in #7107, get a landscape image that will work for one of the planets using the duplicate image (bearing in mind the planet description), and then open a PR that adds the new landscape image and updates the planet to use the newly added image. 

You can refer to #8550 to see what the PR should contain.
